app/presenters/concerns/assetted_view.rb

```diff
@@ -19,6 +19,10 @@ def js_vars
     [
       {
         name: 'pageName', value: page_name
+      },
+      {
+        name: 'requirementsApplication',
+        value: '/assets/application.js'
```

Comment:
Will this work in a production environment with precompiled assets?

Reply:
should work now, it's using the asset_path helper which includes the proper name including the content hash.